src/components/confirmation-dialogs.js

```diff
@@ -19,24 +19,23 @@ export function confirmSectionDelete(t, section, onOk, onCancel = () => {}) {
 
 export function confirmSectionHardDelete(
   t,
-  section,
   onOk,
   onCancel = () => {}
 ) {
   let dialog = null;
-  let deletionReason = '';
-  let deleteDescendants = false;
-  let handleDeletionReasonChange = null;
-  let handleDeleteDescendantsChange = null;
+  let reason = '';
+  let deleteAllRevisions = false;
+  let handleReasonChange = null;
+  let handleDeleteAllRevisionsChange = null;
```

Comment:
Small thing, we could inline these variables, I think there's no need for them to be instantiated with null.

Reply:
Do you mean to not initialize them with default values? Because to inline would mean to me moving their declaration to where they're used (which is not possible as they're used in multiple places).